department-of-veterans-affairs/abd-vro
# Update Logging for Dropdown Conditions

## What was the problem?

After deploying the redesigned dropdown list in va.gov, CC team needs to update the logging values to track usage of the new dropdown options. 

Associated tickets or Slack threads:
- [#322](https://app.zenhub.com/workspaces/contention-classification-640a24db5dcd08457cebbc79/issues/gh/department-of-veterans-affairs/vagov-claim-classification/322)

## How does this fix it?[^1]

This adds a function to build the list of dropdown options from the most current contention classification look up table.

## How to test this PR
- from domain-cc directory run `pytest`
- Step 2[^secrel]


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
